code/stable/glassbr/SRS/GlassBR_SRS.tex

```diff
@@ -1066,13 +1068,13 @@ \subsubsection{Data Definitions}
               \item{$LSF$ is the load share factor (Unitless)}
               \end{symbDescription}
 \\ \midrule \\
-Notes & $LR$ is the load resistance, which is also called capacity. $NFL$ is the non-factored load, as defined in \hyperref[DD:nFL]{DD: nFL}. $GTF$ is the glass type factor, as defined in \hyperref[DD:gTF]{DD: gTF}.
+Notes & $LR$ is also called capacity.
         
-        \hyperref[assumpGL]{A: glassLite}
+        $NFL$ is defined in \hyperref[DD:nFL]{DD: nFL}.
         
-        \hyperref[DD:gTF]{DD: gTF}
+        $GTF$ is defined in \hyperref[DD:gTF]{DD: gTF}.
         
-        \hyperref[DD:nFL]{DD: nFL}
+        The glass is considered to be a single lite (from \hyperref[assumpGL]{A: glassLite}).
```

Comment:
Where did this come from?  It is not in the previous version, and it is not in the case study.  Moreover, this equation has LSF as part of its definition.  The equation does not assume one lite of glass, even though for the overall problem we can make this assumption.  The assumption is not true for this equation.  The assumption is applied elsewhere.

Reply:
The assumption says that LSF = 1 for all equations, which is where its value comes from. Should the note be modified to make this more clear? Something along the lines of "LSF comes from [A: glassLite]"?

Reply:
@samm82, you are missing the point.  This equation does not assume that LSF = 1.  If it did, the equation would be LR = NFL*GTF.  The equation shows LR = NFL*GTF*LSF because in the scope of this definition, LSF can have values other than 1.  When this DD is applied elsewhere, the value of LSF = 1 is substituted in, but that hasn't happened yet.  This should be more clear in the future when we add the derivations.

Reply:
Yup, you're right. I had this thought, but somewhere between thinking "oh yeah, that makes sense" and typing up a response, I confused myself. Should I just remove the assumption? I don't think I should be adding derivations at this stage of the summer. 😂

Reply:
Yes, please remove the assumption from the definition for LR.  The derivations were more something I was thinking about for the future, not this summer.  :-)

Reply:
Done in 4130d37c1.